app/services/registries/validate_service.rb

```diff
@@ -17,14 +17,13 @@ def registry_params
     end
 
     def check_reachability!
-      if only_params.nil? || only_params.include?("hostname")
-        reachable_msg = @registry.reachable?
+      return unless only_params.nil? || only_params.include?("hostname")
```

Comment:
Since you've changed to `unless`, I think you should change from `||` to `&&` to keep the equivalent condition.

Reply:
I think it's right, I got confused. It first checks the condition and then invert with not, so it's still equivalent.